src/firefly/java/edu/caltech/ipac/visualize/plot/FitsRead.java

```diff
@@ -988,30 +927,34 @@ public synchronized void doStretch(RangeValues rangeValues,
                                        int startPixel,
                                        int lastPixel,
                                        int startLine,
-                                       int lastLine, ImageMask[] lsstMasks){
-
-
-
-
-        Histogram hist= getHistogram();
-
-
-        double slow = getSlow(rangeValues, float1d, imageHeader, hist);
-        double shigh = getShigh(rangeValues, float1d, imageHeader, hist);
-        if (SUTDebug.isDebug()) {
-            printInfo(slow, shigh, imageHeader.bitpix, rangeValues);
-        }
+                                       int lastLine, ImageMask[] lsstMasks)  {
```

Comment:
Yes, I agree and will make the corresponding changes.   Since it worked OK, I can do a clean up to remove everything added as testing the mask layer.